# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/contextualidentity/test/browser/browser_saveLink.js
# Commit: 89e396b2896e
# Full Hash: 89e396b2896ed7fc168a5b4ec2f8ead05f5c6802
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2020-06-03 04:20:58
# Regressor Bug: 1638358
# File Overlap Count: 1
# Description:
#   Bug 1638358 - Cookie Schemeful Same-Site - part 5 - schemeful comparison, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75629
# ==============================================================================

diff -r 33ad5fa05209 -r 89e396b2896e browser/components/contextualidentity/test/browser/browser_saveLink.js
--- a/browser/components/contextualidentity/test/browser/browser_saveLink.js	Tue Jun 02 09:14:09 2020 +0000
+++ b/browser/components/contextualidentity/test/browser/browser_saveLink.js	Tue Jun 02 09:15:24 2020 +0000
@@ -13,7 +13,13 @@
 
   // make sure userContext is enabled.
   await SpecialPowers.pushPrefEnv({
-    set: [["privacy.userContext.enabled", true]],
+    set: [
+      ["privacy.userContext.enabled", true],
+      // This test does a redirect from https to http and it checks the
+      // cookies. This is incompatible with the cookie sameSite schemeful
+      // feature and we need to disable it.
+      ["network.cookie.sameSite.schemeful", false],
+    ],
   });
 });
 